### DISCUSSION TITLE
Fee bumping/RBF ETA

### DISCUSSION THREAD

**Discussion by goddesslens** [ordinals/ord#1784] [General]
Creating this discussion because a sub 900 inscirption of mine is stuck at 1 sat/vbyte send. And the buyer has already paid me BTC for it. Are any ord devs working on it for ord wallet ? Any ETA ? Because I don't think mempool will ever clear it as we see more ordinal marketplaces. Also those trying, Bitcoin-cli fee bump command, be careful as your inscription might get spent as fee.

**Comment by ghost** [comment]
> Creating this discussion because a sub 900 inscirption of mine is stuck at 1 sat/vbyte send. And the buyer has already paid me BTC for it.

Buyer can do CPFP for this transaction:

1. Carefully select unconfirmed UTXO associated with this tx
2. Spend it in a transaction and use a new address from your own wallet as output
3. Use fee rate about 30 sat/vbyte

**Reply by goddesslens** [reply]
> > Creating this discussion because a sub 900 inscirption of mine is stuck at 1 sat/vbyte send. And the buyer has already paid me BTC for it.
> 
> Buyer can do CPFP for this transaction:
> 
> 1. Carefully select unconfirmed UTXO associated with this tx
> 2. Spend it in a transaction and use a new address from your own wallet as output
> 3. Use fee rate about 30 sat/vbyte

Possible but buyer(reciever) has inscriptions on his wallet. If he does CPFP, chances are his inscription might be spent as fee. He has to clear his wallet of all inscriptions but that's a headache. So if any ord wallet dev has prioritised RBF in ord wallet itself making sure no inscriptions are spent then that would be great. Taking high precautions as we are in nascent stage and inscriptions are worth a lot in the future.

**Reply by ghost** [reply]
Only selected UTXO will be spent and used for fee + output. Sparrow wallet even shows transaction view before sending.

**Reply by reinamora137** [reply]
> > Creating this discussion because a sub 900 inscirption of mine is stuck at 1 sat/vbyte send. And the buyer has already paid me BTC for it.
> 
> Buyer can do CPFP for this transaction:
> 
> 1. Carefully select unconfirmed UTXO associated with this tx
> 2. Spend it in a transaction and use a new address from your own wallet as output
> 3. Use fee rate about 30 sat/vbyte

Do you have further examples / detail on how to accomplish these steps? Thanks!

**Reply by goddesslens** [reply]
> Only selected UTXO will be spent and used for fee + output. Sparrow wallet even shows transaction view before sending.

Buyer has imported wallet to sparrow (BIP39 taproot) but he is unable to see the unconfirmed transaction. Any guide on how to do this ?

**Reply by ghost** [reply]
Sparrow wallet has a support group on telegram to help with these issues: https://t.me/SparrowWallet
